teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(age, specialization, name = 'Unknow')
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] parent_permission should be passed to super as well otherwise it will always be "true" even if the user enters false